presentation/components/public/locales/en/PresentationComponents.json

```diff
@@ -15,7 +15,10 @@
     "select-instance": "Select element"
   },
   "instance-filter-builder": {
-    "classes": "Classes"
+    "classes": "Classes",
+    "category": "Category: ",
```

Comment:
Is the space at the end of the string is really needed? Can the spacing be done using css as these strings are rendered in a table?